global-search/src/app.js

```diff
@@ -0,0 +1,132 @@
+     * @return {undefined}
+     * */
+    filterResultsTemplate(){
+        var label;
+        var labelFromSession = sessionStorage.getItem('fcLabel');
+        var profileFromSession = sessionStorage.getItem('fcProfile');
+        var scopeFromSession = sessionStorage.getItem('fcScope');
+
+        if(labelFromSession){
+            label = this.capitalizeFirstLetter(labelFromSession);
+        } else if(scopeFromSession) {
+            label = "Results from <strong>" + scopeFromSession + "</strong>";
+        } else if (profileFromSession) {
+            label = this.capitalizeFirstLetter(profileFromSession);
+        }
+
+        var filterResultsTemplate = `
+               <div class="qg-filter-by-results">
+                  <p class="qg-filter-by-results__title">Filter results by</p>
+                  <form class="form qg-forms-v2 qg-filter-by-results__form">
+                  <ol class="questions pt-2">
+                    <li>
+                      <fieldset>
+                        <legend>
+                          <span class="label">Content type</span>
+                        </legend>
+                        <ol class="choices qg-forms-v2__radio">
+                          <li>
+                            <input checked name="filterBy" id="customOption" type="radio" value="custom"
+                                   data-scope="${scopeFromSession}" 
+                                   data-profile="${profileFromSession}"
+                                   data-label="${labelFromSession}" />
+                            <label for="customOption">${label}</label>
+                          </li>
+                          <li>
+                            <input name="filterBy" id="qld" type="radio" value="qld" data-profile='qld' data-scope=''/>
+                            <label for="qld">all Queensland Government</label>
+                          </li>
+                        </ol>
+                      </fieldset>
+                    </li>
+                  </ol>
+                </form>
+                <button type="button" class="qg-btn qg-btn__filter btn-primary mb-2" id="applyFilter">Apply filters</button>
+               </div>
+            `;
+
+        if(this.filterParameter || this.profileParameter !== 'qld' || this.scopeParameter){
+            $('.qg-aside').append(filterResultsTemplate);
+        }
+    }
+
+    /**
+     * trackRadioButtonChanges function track radio button changes when a user interacts with the form to set a session
+     * @return {undefined}
+     * */
+    trackRadioButtonChanges() {
+        var  selectedRadioBtnFromSession = sessionStorage.getItem('selectedRadiobutton');
+        $('input[type=radio][name=filterBy]').on('change', function() {
+            var selectedVal = $(this).val();
+            switch (selectedVal) {
+                case 'qld':
+                    sessionStorage.setItem(`selectedRadiobutton`, selectedVal);
```

Comment:
for consistency could use `fc` as prefix for the key
ie. `fcSelectedRadiobutton `

Reply:
updated in the recent commit